docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -87,19 +90,26 @@ The following example shows how to create a gallery-like flexbox layout using th
     content.JustifyContent = FlexContainer.Justification.JustifyCenter;
     /// Align items vertically center
     content.AlignItems = FlexContainer.Alignment.AlignCenter;
-    /// Occupy 90 percent of available space on the cross axis
-    content.Flex = 0.9f;
     /// Add the content area to the main container
     flexContainer.Add(content);
     ```
 
-7.  Add items to the toolbar.
+7.  Use `Flex` property of the `FlexContainer` class to specify the area to be occupied within the main container. Since the main container displays items vertically its height is shared between the toolbar and the content. Setting `toolbar.Flex` to 0.1 and `content.Flex` to 0.9 means that `toolbar` will occupy 10% of vertical space and `content` will occupy 90% of vertical space of the main container. Width of items will be eqaul to width of main container.
```

Comment:
```suggestion
7.  Use `Flex` property of the `FlexContainer` class to specify the area to be occupied within the main container. Since, the main container displays items vertically, its height is shared between the toolbar and the content. Setting `toolbar.Flex` to 0.1 and `content.Flex` to 0.9 means that `toolbar` will occupy 10% of vertical space and `content` will occupy 90% of vertical space of the main container. Thus, the width of the items equals the width of the main container:

Reply:
I have two questions : 

> Since,

I'm not sure with the utility of adding comma after "Since". How about not adding the comma after "Since"?

>Thus, the width 

I'm not sure with the "Thus" here. The previous sentence is talking about **vertical height**, so why "Thus" is added to **width issue**?

Reply:
@mijin85cho both are correct, I took your suggestion as it looks fine. 
Why I had added (,) was to give a slight pause. 
I had added "thus" to give a notion of inference.